nbs/core.ipynb

```diff
@@ -3204,6 +3321,147 @@
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0441e9d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
+    "# test conformal prediction, method=conformal_distribution\n",
+    "\n",
+    "conformal_intervals = ConformalIntervals()\n",
+    "\n",
+    "models = []\n",
+    "for nf_model in [NHITS, RNN, StemGNN]:\n",
+    "    params = {\"h\": 12, \"input_size\": 24, \"max_steps\": 1}\n",
+    "    if nf_model.__name__ == \"StemGNN\":\n",
+    "        params.update({\"n_series\": 2})\n",
+    "    models.append(nf_model(**params))\n",
+    "\n",
+    "\n",
+    "nf = NeuralForecast(models=models, freq='M')\n",
+    "nf.fit(AirPassengersPanel_train, conformal_intervals=conformal_intervals)\n",
+    "preds = nf.predict(futr_df=AirPassengersPanel_test, conformal_level=[10, 50, 90])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9eb89f3b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
+    "#| polars\n",
+    "# test conformal prediction works for polar dataframe\n",
+    "\n",
+    "conformal_intervals = ConformalIntervals()\n",
+    "\n",
+    "models = []\n",
+    "for nf_model in [NHITS, RNN, StemGNN]:\n",
+    "    params = {\"h\": 12, \"input_size\": 24, \"max_steps\": 1}\n",
+    "    if nf_model.__name__ == \"StemGNN\":\n",
+    "        params.update({\"n_series\": 2})\n",
+    "    models.append(nf_model(**params))\n",
+    "\n",
+    "\n",
+    "nf = NeuralForecast(models=models, freq='1mo')\n",
+    "nf.fit(AirPassengers_pl, conformal_intervals=conformal_intervals, time_col='time', id_col='uid', target_col='target')\n",
+    "preds = nf.predict(conformal_level=[10, 50, 90])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0db88cac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
+    "# test conformal prediction, method=conformal_error\n",
+    "\n",
+    "conformal_intervals = ConformalIntervals(method=\"conformal_error\")\n",
+    "\n",
+    "models = []\n",
+    "for nf_model in [NHITS, RNN, StemGNN]:\n",
+    "    params = {\"h\": 12, \"input_size\": 24, \"max_steps\": 1}\n",
+    "    if nf_model.__name__ == \"StemGNN\":\n",
+    "        params.update({\"n_series\": 2})\n",
+    "    models.append(nf_model(**params))\n",
+    "\n",
+    "\n",
+    "nf = NeuralForecast(models=models, freq='M')\n",
+    "nf.fit(AirPassengersPanel_train, conformal_intervals=conformal_intervals)\n",
+    "preds = nf.predict(futr_df=AirPassengersPanel_test, conformal_level=[10, 50, 90])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d25b2cd2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
+    "# test conformal prediction are not applied for models with quantiled-related loss\n",
```

Comment:
This test isn't necessary when removing `enable_quantiles` argument. It's fine to apply conformal to any quantile loss.